geoips/commandline/log_setup.py

```diff
@@ -16,6 +16,21 @@
 import sys
 
 
```

Comment:
Overall this looks good. I'd like to see one more thing, though. Can you try implementing a unit test for this? It would go in `tests/unit_tests/commandline/log_setup.py` and should make use of pytest's `caplog` feature to make sure the output is what it should be.

I'm asking for this for a few reasons. First, to get in the habit of writing unit tests for everything as we go. Second, to test the `caplog` feature to see if we should apply it elsewhere. Third, because the change asked for in the other conversation will make this more complicated in a way that should likely be tested.

Here's some info on `caplog`: https://pytest-with-eric.com/fixtures/built-in/pytest-caplog/

I think you'd basically just want to test the number of asterisks and number of lines given different length inputs.

Reply:
This looks good. Thanks @evrose54!